xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.h

```diff
@@ -235,7 +235,7 @@ class CWinRenderer : public CBaseRenderer
   unsigned int         m_destHeight;
 
   int                  m_neededBuffers;
-  unsigned int         m_frameIdx;
+  unsigned int         m_frameIdx = 0;
```

Comment:
This seemed to be the default value used by CWinRenderer::SelectRenderMethod(): https://github.com/xbmc/xbmc/blob/8405bbf242a73e7b88641c7db4a9808fd46a2f47/xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.cpp#L221